datasource/src/main/scala/quasar/blobstore/services.scala

```diff
@@ -17,17 +17,19 @@
 package quasar.blobstore
 
 import slamdata.Predef._
-import quasar.api.resource.{ResourceName, ResourcePath, ResourcePathType}
+import quasar.blobstore.paths.{BlobPath, BlobstorePath, PrefixPath}
 
+import cats.data.Kleisli
 import fs2.Stream
 
-trait Blobstore[F[_]] {
+object services {
 
-  def list(path: ResourcePath): F[Option[Stream[F, (ResourceName, ResourcePathType)]]]
+  type StatusService[F[_]] = F[BlobstoreStatus]
 
-  def get(path: ResourcePath): Stream[F, Byte]
+  type GetService[F[_]] = Kleisli[F, BlobPath, Option[Stream[F, Byte]]]
```

Comment:
I think it would be good to add a `BlobstoreError` class and then change these `Option[X]`s into `Either[BlobstoreError, X]`s (in a next PR)